eng/scripts/smoketest/Get-Package-Version.ps1

```diff
@@ -23,7 +23,10 @@ function Get-SmokeTestPkgProperties
 
     $NugetSource="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v2"
     $allPackages = Get-AllPkgProperties
-    $newPackages = $allPackages.Where({ $_.IsNewSdk })
+    $excludePackages = @{
+        "Azure.AI.Personalizer" = 'Unable to load DLL rl.net.native.dll https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1492856&view=ms.vss-test-web.build-test-results-tab&runId=30630890&resultId=100012&paneView=debug';
```

Comment:
Is there an issue filed for this? If so please link the issue here and if not please file an issue at put this context in it and link it here.

Reply:
I would just paste the full error message in the comment instead of adding a link to an internal pipeline:

```
System.TypeInitializationException : The type initializer for 'Rl.Net.Native.NativeMethods' threw an exception.
----> System.DllNotFoundException : Unable to load DLL 'rl.net.native.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E)
```

Reply:
We can link to the internal pipeline for debugging in the issue description after it's created.